Functions.Templates/Templates/SqlOutputBinding-PowerShell/run.ps1

```diff
@@ -15,14 +15,8 @@ param($Request)
 # Write to the Azure Functions log stream.
 Write-Host "PowerShell SQL Binding function processed a request."
 
-# Set results array to req_body for upsertion
-# Note that this expects the body to be a JSON object or array of objects which has a property 
-# matching each of the columns in the table to upsert to.
-$req_body = $Request.Body | ConvertTo-Json
+# Update req_body with the body of the request
+$req_body = @{'Insert Values Here'="Insert Value Here"} | ConvertTo-Json
 
 # Associate values to output bindings by calling 'Push-OutputBinding'.
-Push-OutputBinding -Name Response -Value ([HttpResponseContext]@{
```

Comment:
And keep the HttpResponse with the $req_body value to match our other templates.